source/admin_manual/list_of_events.rst

```diff
@@ -1792,581 +1093,421 @@ Events emitted by Dovecot's internal SQL library.
 
 .. Note:: This includes queries sent to Cassandra.
 
+.. dovecot_core:event:: sql_query_finished
+
+   :field error: Human readable error.
+   :field error_code: Error code (if available).
+   :field query_first_word: First word of the query (e.g. ``SELECT``).
 
-sql_query_finished
-------------------
+   TODO
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| error               | Human readable error                                 |
-+---------------------+------------------------------------------------------+
-| error_code          | Error code (if available)                            |
-+---------------------+------------------------------------------------------+
-| query_first_word    | First word of the query (e.g. ``SELECT``)            |
-+---------------------+------------------------------------------------------+
 
+.. dovecot_core:event:: sql_transaction_finished
 
-sql_transaction_finished
-------------------------
+   :field error: Human readable error.
+   :field error_code: Error code (if available).
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| error               | Human readable error                                 |
-+---------------------+------------------------------------------------------+
-| error_code          | Error code (if available)                            |
-+---------------------+------------------------------------------------------+
+   TODO
 
 
-sql_connection_finished
------------------------
+.. dovecot_core:event:: sql_connection_finished
+
+   TODO
```

Comment:
Connection to SQL server is closed. I don't know if this event is of any use.

Reply:
I think it theoretically might be useful to debug/monitor how often Dovecot is connecting to the external service.  Regardless, this seems small and straightforward enough that I don't see much utility in removing it.